app/controllers/laboratory_samples_controller.rb

```diff
@@ -36,9 +36,28 @@ def create
     end
   end
 
+  def edit
+    @sample = LaboratorySample.find(params[:id])
+    @can_delete = true
```

Comment:
Let's avoid adding options in the UI before allowing the user to perform that action.

If, as of this commit, we don't support sample deletion, let's delay the `@can_delete = true` to whenever we have implemented the deletion.

We're merging all these related PRs in a row, but commits should be independent from each other - if we merge this PR without merging any other (for whatever reason), things should not fail.